chainstate/src/detail/tests/processing_tests.rs

```diff
@@ -531,11 +533,15 @@ fn test_consensus_type() {
             btf.get_block(btf.block_indexes[i - 1].get_block_id().clone()).unwrap().unwrap();
         let mut mined_block = btf.random_block(&prev_block, None);
         let bits = min_difficulty.into();
-        assert!(
-            crate::detail::pow::work::mine(&mut mined_block, u128::MAX, bits, vec![])
-                .expect("Unexpected conversion error")
-        );
-        assert!(btf.add_special_block(mined_block).is_ok());
+        let (_, pub_key) = PrivateKey::new(KeyKind::RistrettoSchnorr);
+        assert!(crate::detail::pow::work::mine(
+            &mut mined_block,
+            u128::MAX,
+            bits,
+            vec![TxOutput::new(Amount::from_atoms(10), Destination::PublicKey(pub_key))]
+        )
+        .expect("Unexpected conversion error"));
+        assert!(btf.add_special_block(mined_block).unwrap().is_some());
```

Comment:
Please just unwrap. Do not test for is_some or is_ok.

Reply:
Ok, fixed!